packages/core/src/DropDownMenu/DropDownMenu.stories.tsx

```diff
@@ -33,74 +27,47 @@ export const Main: StoryObj<HvDropDownMenuProps> = {
     placement: "left",
     keepOpened: false,
     disabled: false,
-    expanded: true,
-    defaultExpanded: false,
+    expanded: undefined,
+    defaultExpanded: true,
     category: "secondaryGhost",
   },
   argTypes: {
     classes: { control: { disable: true } },
     onToggle: { control: { disable: true } },
     onClick: { control: { disable: true } },
   },
-  render: ({ expanded, onToggle, ...rest }) => {
-    const [open, setOpen] = useState<boolean>(!!expanded);
-
-    return (
-      <HvDropDownMenu
-        expanded={open}
-        onToggle={() => setOpen((o) => !o)}
-        {...rest}
-      />
-    );
+  render: (args) => {
+    return <HvDropDownMenu {...args} />;
   },
 };
 
+/** wrapper needed for Storybook not to crash */
+const renderIcon = (Icon: React.ElementType) => () => <Icon />;
+
 export const WithIconsAndActions: StoryObj<HvDropDownMenuProps> = {
   parameters: {
     docs: {
       description: {
-        story:
-          "DropDownMenu with Icons and Actions. Icons should be colored accordingly when selected.",
+        story: "DropDownMenu with icons and disabled actions",
       },
     },
-  },
-  render: () => {
-    const iconSelectedColor =
-      (Icon: React.ElementType): HvListValue["icon"] =>
-      ({ isSelected }) =>
-        <Icon color={isSelected ? "atmo1" : undefined} />;
-
-    return (
-      <HvDropDownMenu
-        expanded
-        placement="right"
-        onClick={(e, item) => console.log(item.label)}
-        dataList={[
-          { label: "Label 1", icon: iconSelectedColor(User) },
-          { label: "Label 2", icon: iconSelectedColor(Calendar) },
-          { label: "Label 3", icon: iconSelectedColor(Plane) },
-        ]}
-      />
-    );
-  },
-};
-
-export const DisabledItems: StoryObj<HvDropDownMenuProps> = {
-  parameters: {
-    docs: {
-      description: {
-        story: "DropDownMenu with disabled items.",
+    parameters: {
+      eyes: {
+        runBefore() {
```

Comment:
Looking at Applitools, the dropdown menu doesn't seem to have opened for some reason. Maybe doing `waitFor` the `menu` could solve this.